### PR TITLE
Move fixtures to its own folder, enhance config tests

### PR DESCRIPTION
**Pull request by mudler**
The added test is introduced while trying to debug #223

Signed-off-by: Ettore Di Giacinto <edigiacinto@suse.com>

### CODE REVIEW

File: cmd/config/config_test.go [hunk 1]
```diff
@@ -49,6 +46,30 @@ var _ = Describe("Config", Label("config"), func() {
 	AfterEach(func() {
 		viper.Reset()
 	})
+
+	Context("From fixtures", func() {
+		Describe("read all specs", Label("install"), func() {
+			It("reads values correctly", func() {
+				cfg, err := ReadConfigRun("../../tests/fixtures/simple/", nil, mounter)
```
Comment (Itxaka) [inline cmd/config/config_test.go:53]: blarg, isnt there a way of getting the root of the project somehow? this is really ugly :/

**Review by Itxaka — APPROVED** [review]

File: cmd/config/config_test.go (hunk 1 above)
Comment (mudler) [reply cmd/config/config_test.go:53]: why? This is completely fine with go, as paths are resolved from where the unit test is

Comment (Itxaka) [reply cmd/config/config_test.go:53]: `¯\_(ツ)_/¯` I just dont really like this relative path vs full paths

Comment (mudler) [reply cmd/config/config_test.go:53]: blargh I've tried to have a look but the only way would be to use `runtime.Caller`, but things then look even more uglier..

**Pull request merged**